### ISSUE TITLE
【VC模式】VC模式下的流式推理，已经是分段返回，但合成音频时仍然只有最后一段

### ISSUE THREAD

**Issue by wang-TJ-20** [FunAudioLLM/CosyVoice#675]
VC模式下的流式推理，已经是分段返回，但合成音频时仍然只有最后一段，下面是我的调用代码

`cosyvoice = CosyVoice("tts_weight/CosyVoice-300M-25Hz")
prompt_speech_16k = load_wav('girl_cut.wav', 16000)
source_speech_16k = load_wav('AP.wav', 16000)

audio_tts = []
for i, j in enumerate(cosyvoice.inference_vc(source_speech_16k, prompt_speech_16k, stream=False)):
    audio_tts.append(j["tts_speech"])
full_tts = torch.cat(audio_tts, dim=1)
torchaudio.save("vc_{}.wav".format(2222), j["tts_speech"], 22050)`

**Comment by aluminumbox** [comment]
你这里stream=False

**Comment by wang-TJ-20** [comment]
@aluminumbox 嗯嗯，解决啦，我代码有个bug
